sztp-agent/pkg/secureagent/agent.go

```diff
@@ -68,6 +75,11 @@ type BootstrapServerErrorOutput struct {
 	} `json:"ietf-restconf:errors"`
 }
 
+type HttpClient interface {
```

Comment:
let's move this from `agent` to `tls` ?

Reply:
ok, so you want interface here - fine
but then the object itself - http or ftp in their own files... ok ?

Reply:
Yes exactly.Have separate files for each client